src/lib_midi_channel.c

```diff
@@ -0,0 +1,1082 @@
+#include "global.h"
```

Comment:
perhaps the midi source files could go in their own src subfolder (and with shorter filenames)

Reply:
I was also under the impression that each external library should be in their own respective subfolder, though I'll keep to the current topic and just put these ones in `src/midi/`.

Reply:
yeah organizing libraries into subfolders is definitely nice